Error en la petici√≥n POST: 400
Mensaje de error: {
  "validation_error": {
    "body_params": [
      {
        "input": {
          "cellphone_number": "3178696749",
          "country": 99,
          "country_company": 99,
          "country_indicative": 57,
          "email": "kacey_yundt@yopmail.com",
          "identity_number_company": "900691928",
          "name": "Amber",
          "surname": "D'Amore"
        },
        "loc": [
          "password"
        ],
        "msg": "Field required",
        "type": "missing",
        "url": "https://errors.pydantic.dev/2.8/v/missing"
      },
      {
        "input": {
          "cellphone_number": "3178696749",
          "country": 99,
          "country_company": 99,
          "country_indicative": 57,
          "email": "kacey_yundt@yopmail.com",
          "identity_number_company": "900691928",
          "name": "Amber",
          "surname": "D'Amore"
        },
        "loc": [
          "post"
        ],
        "msg": "Field required",
        "type": "missing",
        "url": "https://errors.pydantic.dev/2.8/v/missing"
      }
    ]
  }
}
